src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/elevator/ElevatorPlate.java

```diff
@@ -96,7 +97,7 @@ public BlockUseHandler getItemHandler() {
         LinkedList<ElevatorFloor> floors = new LinkedList<>();
         int index = 0;
 
-        for (int y = 0; y < b.getWorld().getMaxHeight(); y++) {
+        for (int y = WorldUtils.getMinHeight(b.getWorld()); y < b.getWorld().getMaxHeight(); y++) {
```

Comment:
A small thing, but you could probably create variables for these calls to improve performance a little bit

Reply:
This only calls once so no saving would be achieved